### DISCUSSION TITLE
Password link

### DISCUSSION THREAD

**Discussion by fridayshoes** [GerryDazoo/Slinger#107] [Q&A]
Is the password link mentioned on the setup page likely to stop working once we get to November?

**Comment by GerryDazoo** [comment]
Yes.

**Reply by donscreen** [reply]
I also found my admin passwords in the console log for Slingplayer Desktop...under appdata and Sling Media.

**Reply by fridayshoes** [reply]
Does this mean that anyone who doesn't note down the info for their password link now will permanently be unable to use their slingbox once this information is lost?

**Reply by GerryDazoo** [reply]
Thanks for the info donscreen. Very useful. So we will be able to get current passwords after the servers are down.
Fridayshoes: So the answer to your question in no. You will be able to use them based on donscreens discovery. We also know now what the default password is for the box after a factory reset.  So all is not lost.

**Reply by donscreen** [reply]
This is where I found the admin passwords on my Windows file search:
"C:\Users\donaldscreen\AppData\Roaming\Sling Media\Slingplayer Desktop\Logs\console.log"

**Reply by sp222222** [reply]
donscreen - thanks for this folder search. I don't have slingplayer desktop but I do use slingfront and found ALL nine of the slingboxes I have ever owned in here ! it has the password "encrypted" and the finderID AND the channel codes  in plain text !!

AppData\Roaming\SlingFront\SlingBoxes

**Comment by f520k510** [comment]
Is it just me or not?  I found the console.log file but the password seemed to be encrypted.  It's different from those in the sling box url.

**Reply by GerryDazoo** [reply]
do they look something like this "69om9nwt008i7wm"? Then yes, it's not encrypted.

**Reply by f520k510** [reply]
The password to my 500 looks like this in my console.log:
"password":"E1:7b9265efa844944bea6461ffadf1773d215779716fbb5c330875785b9b9c1635"

**Reply by GerryDazoo** [reply]
I found the clear test password in this log entry

4:46:4 215 : 5.0.0.83 : INFO :: DeLoreanLib :: Utils :: setConnectionParamsToConfig...connection Object = {"id":"c631844cf70531865d73650a16a0a536","finderId":"c631844cf70531865d73650a16a0a536","productId":18,"isBlocked":false,"isSaved":1,"password":**"MySlingboxPassword"**,"name":"My Slingbox","passAutoMode":true,"memberId":3568457,"lookupByFId":true,"isOwner":true,"isConfigured":1,"isAP":false,"status":"configured","btnText":"Reconfigure","isAvailable":1,"username":"admin","isDiscovered":1,"displayStatus":"available","$$hashKey":"020"}

**Reply by f520k510** [reply]
Can you tell me which version of slingplayer desktop you have?
I have the latest 5.0.0.83 and I am worried it is why my passwords in consoles.log are different from those I found with the url in the readme.txt of slinger.  Another big difference is that I am using windows 10 traditional Chinese version.
